src/tty/windows.rs

```diff
@@ -0,0 +1,285 @@
+        interest: mio::Ready,
+        poll_opts: mio::PollOpt,
+    ) -> io::Result<()> {
+        self.read_token = (*token.next().unwrap()).into();
+        self.write_token = (*token.next().unwrap()).into();
+        if interest.is_readable() {
+            poll.register(
+                &self.conout,
+                self.read_token,
+                mio::Ready::readable(),
+                poll_opts,
+            )?
+        } else {
+            poll.register(
+                &self.conout,
+                self.read_token,
+                mio::Ready::empty(),
+                poll_opts,
+            )?
+        }
+        if interest.is_writable() {
+            poll.register(
+                &self.conin,
+                self.write_token,
+                mio::Ready::writable(),
+                poll_opts,
+            )?
+        } else {
+            poll.register(
+                &self.conin,
+                self.write_token,
+                mio::Ready::empty(),
+                poll_opts,
+            )?
+        }
+        Ok(())
+    }
+
+    #[inline]
+    fn reregister(&mut self, poll: &mio::Poll, interest: mio::Ready, poll_opts: mio::PollOpt) -> io::Result<()> {
+        if interest.is_readable() {
+            poll.reregister(
+                &self.conout,
+                self.read_token,
+                mio::Ready::readable(),
+                poll_opts,
+            )?;
+        } else {
+            poll.reregister(
+                &self.conout,
+                self.read_token,
+                mio::Ready::empty(),
+                poll_opts,
+            )?;
+        }
+        if interest.is_writable() {
+            poll.reregister(
+                &self.conin,
+                self.write_token,
+                mio::Ready::writable(),
+                poll_opts,
+            )?;
+        } else {
+            poll.reregister(
+                &self.conin,
+                self.write_token,
+                mio::Ready::empty(),
+                poll_opts,
+            )?;
+        }
+        Ok(())
+    }
+
+    #[inline]
+    fn deregister(&mut self, poll: &mio::Poll) -> io::Result<()> {
+        poll.deregister(&self.conout)?;
+        poll.deregister(&self.conin)?;
+        Ok(())
+    }
+
+    #[inline]
+    fn reader(&mut self) -> &mut NamedPipe {
+        &mut self.conout
+    }
+
+    #[inline]
+    fn read_token(&self) -> mio::Token {
+        self.read_token
+    }
+
+    #[inline]
+    fn writer(&mut self) -> &mut NamedPipe {
+        &mut self.conin
+    }
+
+    #[inline]
+    fn write_token(&self) -> mio::Token {
+        self.write_token
+    }
+}
+
+impl<'a> OnResize for Winpty<'a> {
+    fn on_resize(&mut self, sizeinfo: &SizeInfo) {
+        if sizeinfo.cols().0 > 0 && sizeinfo.lines().0 > 0 {
+            self.set_size(sizeinfo.cols().0, sizeinfo.lines().0)
+                .unwrap_or_else(|_| info!("Unable to set winpty size, did it die?"));
+        }
+    }
+}
```

Comment:
Something went wrong here during rebase probably.

Reply:
Damn it, I thought I was a skilled guy user :(